dev/com.ibm.ws.kernel.service/src/com/ibm/ws/kernel/service/util/CpuInfo.java

```diff
@@ -133,10 +144,10 @@ private synchronized double getSystemCPU() {
         if (cpuUsage >= 0) {
             cpuUsage *= 100;
             if (cpuUsage > 100) {
+                cpuUsage = -1;
                 if (tc.isEventEnabled()) {
                     Tr.event(tc, "getSystemCPU error", ("system CPU out-of-range: " + cpuUsage));
                 }
```

Comment:
I assume you added the cpuUsage = -1 back in because of a problem you hit.  Setting it to -1 here instead of where it was set before causes the trace statement not have the actual out or range value.  I updated it to move it down, but also to only set it to -1 if it is greater than 105.
```suggestion
                if (tc.isEventEnabled()) {
                    Tr.event(tc, "getSystemCPU error", ("system CPU out-of-range: " + cpuUsage));
                }
                if (cpuUsage > 105) {
                    cpuUsage = -1;
                }
```

Reply:
Initially I took "cpuUsage = -1" out of they systemCpu calculation accidentally - I meant to take it out of processCpu - and when I put it back in, I accidentally put it in the wrong spot relative to the trace ... good catch. 

But I thought we were changing this only for processCpu - systemCpu over 100% does seem "wrong", doesn't it?

Reply:
Makes sense.